x-pack/plugins/viz_editor/public/pseudo_plugins/indexpattern_datasource_plugin/to_expression.ts

```diff
@@ -78,6 +78,17 @@ function queryToEsAggsConfigs(query: Query): any {
   });
 }
 
+function fixBucketAggsWithoutMetricColumn(aggs: any[]) {
+  const onlySegments = aggs.every(agg => agg.schema === 'segment');
+  if (onlySegments) {
+    return [
+      ...aggs,
+      { enabled: true, id: String(aggs.length), params: {}, schema: 'metric', type: 'count' },
```

Comment:
This is a hack just for now - esaggs doesn't support agg configs consisting only out of segments, but it is a valid use case IMHO. Should be fixed in esaggs in the long run

Reply:
Yeah, seems like it should be a valid query